src/components/FilterCompany/FilterCompany.js

```diff
@@ -0,0 +1,56 @@
+import React, {useState} from 'react'
+import './FilterCompany.css'
```

Comment:
Можно использовать модульный подход и импортировать объект стилей.
```suggestion
import styles from './FilterCompany.module.css'
```